### DISCUSSION TITLE
Want CRUD in modal.

### DISCUSSION THREAD

**Discussion by Hiren-Gamit** [Laravel-Backpack/community-forum#859] [Q&A (Help)]
I want to some CRUD operation in bootstrap modal. 
I want this feature because in case i want to add and show some details from any other page so i can click the button and opan modal. In that modal first list and at the top there are one button to create new and in list edit delate any other operation through modal without loading page.

**Comment by jcastroa87** [comment]
Hello @Hiren-Gamit, if you're looking to add a new entry within a modal without closing the current one, you can utilize the [InlineCreate Operation](https://backpackforlaravel.com/docs/6.x/crud-fields#create-related-entries-in-a-modal-using-the-inlinecreate-operati). However, for a full CRUD operation within a modal, we currently don't offer that feature. Implementing a comprehensive CRUD within a modal presents development challenges, and at the moment, we're focusing on features that benefit a larger user base. If you make progress on this requirement, feel free to share it with the community. I'll keep this thread open to gather opinions and contributions from other users.

Cheers.

**Reply by Hiren-Gamit** [reply]
@jcastroa87  thanks fore response. 

I tried full CRUD in modal but just one thing is not working in that. That is a if fields and columns CSS and JS is not loading.

**Reply by jcastroa87** [reply]
Hi @Hiren-Gamit, as I indicated previously, although it is a great feature, it is a big thing for the team to start development and make it dynamic enough to meet many requirements. It is within the roadmap developments, however, that will take a few months; while it is more convenient that you develop a particular function that only serves your project, if you achieve something better, it would be great if you could share it here. For now, I will close this thread, but if you need to reopen it or create a new one, you are free to do so.

Cheers.
